@here/harp-mapview/lib/MapView.ts

```diff
@@ -651,16 +654,31 @@ const MapViewDefaults = {
  */
 export interface LookAtParams {
     /**
-     * Target/look at point of the MapView.
+     * Look at point target (or box or set of points of the MapView.
+     *
+     *
      * @note If the given point is not on the ground (altitude != 0) [[MapView]] will do a
      * raycasting internally to find a target on the ground.
+     *
      * As a consequence [[MapView.target]] and [[MapView.zoomLevel]] will not match the values
      * that were passed into the [[MapView.lookAt]] method.
      * @default `new GeoCoordinates(25, 0)` in [[MapView.constructor]] context.
      * @default [[MapView.target]] in [[MapView.lookAt]] context.
      */
     target: GeoCoordLike;
 
+    /**
+     * Setup view, to look at box/ all points.
+     *
+     * If `target` is not specified, then geobox `center` or world box of `points` will be taken
+     * as new `target`.
+     *
+     * Note
+     *  * in sphere some points are not visible if you specify bounds that bigger than 180 degreess
+     *    in latitude
+     */
+    bounds: GeoBox | GeoBoxExtentLike | GeoCoordLike[];
```

Comment:
What would be the center in case of `GeoBoxExtentLike` ?

Reply:
`target` if specified or `MapView.target` if not specified